Added some output for the goroutine leak test. Exactly mimics a fatal error for a regular test (which are sent to stdout), other than the fact that it occurs after they've all passed:

```
$ go test
PASS
--- FAIL: TestMain
	follower_test.go:102: Possible goroutine leak
exit status 1
FAIL	github.com/papertrail/go-tail/follower	20.752s
```

--- a/follower/follower_test.go
+++ b/follower/follower_test.go
@@ -99,7 +99,7 @@ func TestMain(m *testing.M) {
 			// Heuristic to detect leaked goroutines.
 			fmt.Println("--- FAIL: TestMain")
 			logger := log.New(os.Stdout, "\t", log.Lshortfile)
-			logger.Fatal("Possible goroutine leak")
+			logger.Fatal("possible goroutine leak")
 		}
 	}
 	os.Exit(rs)

